crates/bevy_ui/src/render/mod.rs

```diff
@@ -249,11 +249,12 @@ pub fn extract_default_ui_camera_view<T: Component>(
                 .spawn()
                 .insert(ExtractedView {
                     projection: projection.get_projection_matrix(),
-                    transform: GlobalTransform::from_xyz(
+                    transform: Affine3A::from_translation(Vec3::new(
```

Comment:
I feel like I would generally prefer to see `GlobalTransfrom::from_xyz()` as before and keep all of its API, but I'll defer to @cart's judgement.

Reply:
Agreed!

Reply:
Added the construction methods back.